libs/sr/page-team/src/TeammateDisplay/Optimize/index.tsx

```
@@ -89,23 +115,23 @@ export default function Optimize() {
   useEffect(() => () => cancelToken.current(), [])
 
   const onOptimize = useCallback(async () => {
-    if (!optTarget || !calc) return
+    if (!calc) return
     const cancelled = new Promise<void>((r) => (cancelToken.current = r))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)